Auth0.podspec

```diff
@@ -66,16 +75,16 @@ Pod::Spec.new do |s|
   s.ios.dependency 'SimpleKeychain'
   s.ios.dependency 'JWTDecode'
   s.osx.source_files = 'Auth0/*.swift'
-  s.osx.exclude_files = web_auth_files
+  s.osx.exclude_files = [*web_auth_files, *ios_files]
```

Comment:
have you tested array params work?

Reply:
I ran `pod lib lint`, which [builds the spec for all supported platforms](https://github.com/CocoaPods/CocoaPods/issues/8098#issuecomment-422160666).